scripts/publish-biblio.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+
+$(npm bin)/ecmarkup --verbose spec.html --write-biblio biblio/biblio.json /dev/null
+
+cp LICENSE.md biblio
+
+cd biblio
+
+TS=$(date +%s)
+npm version --no-git-tag-version "1.0.${TS}"
+
+SHORT_COMMIT=$(git rev-parse --short HEAD)
+LONG_COMMIT=$(git rev-parse --verify HEAD)
+echo "
+This version was built from commit [${SHORT_COMMIT}](https://github.com/tc39/ecma262/tree/${LONG_COMMIT})." >> README.md
+
+npm publish
```

Comment:
one thought is that this won't be idempotent - it'll either fail on a rerun, or needlessly publish a new one that might not actually be later - so rerunning an old one would cause lots of havoc.

it seems better to find some way for it to be deterministic but also monotonically increasing.

Reply:
Oh, good point. Switched to the timestamp of the commit itself.

Reply:
Normally that wouldn't be any better, but since the policy in this repo is to use `git commit --amend --date=now` before landing everything, we do in fact have this guarantee.

Reply:
Squash-and-merge and rebase-and-merge give the same guarantee, I'm pretty sure; you have to go pretty far out of your way to end up with non-monotonic commit times.

Reply:
True as well.